Source/WebCore/css/CSSFontFeatureValuesRule.idl

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2022 Apple Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU Library General Public License
+ * along with this library; see the file COPYING.LIB.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA.
+ */
+
+typedef USVString CSSOMString;
+
+[
+    Exposed=Window
+] interface CSSFontFeatureValuesRule : CSSRule {
+    readonly attribute sequence<CSSOMString> fontFamilies;
+};
```

Comment:
Huh. That makes this easy, I guess.

Reply:
Indeed this patch doesn't try to expose the children at-rules.
Gecko did the same https://searchfox.org/mozilla-central/source/dom/webidl/CSSFontFeatureValuesRule.webidl

I've added a comment in the IDL file.

Reply:
Oh, is this contrary to the spec?

Can we get the spec changed if both us and Gecko are going to do this?

Reply:
Indeed it would be worth discussing/editing the spec